examples/context/sub/Dockerfile

```diff
@@ -1,6 +1,10 @@
 #use the golang base image
 FROM golang:1.7
-MAINTAINER Daniël van Gils
+LABEL MAINTAINER="Daniël van Gils"
```

Comment:
Oh! I didn't know that it was deprecated almost one and a half years ago. Good catch.

https://github.com/moby/moby/pull/25466

Reply:
I am sorry, I should have added a comment to explain this. Happy to help :)